yarn.lock

```diff
@@ -2994,9 +2994,9 @@ validate-npm-package-license@^3.0.1:
     spdx-correct "~1.0.0"
     spdx-expression-parse "~1.0.0"
 
-vanilla-framework@1.8.1:
-  version "1.8.1"
-  resolved "https://registry.yarnpkg.com/vanilla-framework/-/vanilla-framework-1.8.1.tgz#0d7f5885cdbd1355a9f9d42d7596f9a1258b27ea"
+vanilla-framework@2.0.0-alpha:
```

Comment:
This still has the alpha. ./run the project and then commit the automatic changes that are made to the lockfile

Reply:
Our `./run` the project based on updates. Hopefully, this has fixed the lockfile 🤔

Reply:
@kwm14 I forgot to tell you to `./run clean` first, sorry